src/config/glean.js

```diff
@@ -37,8 +37,12 @@ export const FIREFOX_ON_GLEAN = {
     ping_type: {
       title: 'Ping Type',
       key: 'ping_type',
-      values: [{ key: '*', label: 'All' }],
-      defaultValue: '*',
+      values: [
+        { key: '*', label: 'All' },
+        { key: 'metrics', label: 'Metrics' },
+        { key: 'baseline', label: 'Baseline' },
+      ],
+      defaultValue: 'metrics',
```

Comment:
I assumed that FOG metrics have just one ping type like Fenix metrics -- `metrics` -- but this is not right. I ran a query against the database to check how many ping types are being sent to the ETL now and got 3 total.

```
postgres=# select ping_type from glam_fog_aggregation group by ping_type;
 ping_type 
-----------
 *
 metrics
 baseline
(3 rows)
```